src/app/inventory/store/d1-item-factory.ts

```diff
@@ -472,21 +472,6 @@ function makeItem(
     createdItem.equippingLabel = undefined;
   }
 
-  // do specific things for specific items
-  if (createdItem.hash === 491180618) {
-    // Trials Cards
```

Comment:
Yes, special rendering for Trials cards in D1 is a thing of the past.

Reply:
It does not matter as trials is not available in D1 anymore.